### DISCUSSION TITLE
How to disable the typewriter animation?

### DISCUSSION THREAD

**Discussion by levid0s** [TheR1D/shell_gpt#172] [Q&A]
The `--no-animation` option is no longer present.

**Comment by xsoc** [comment]
I believe it's now a real time stream of the output from GPT, rather than a faked animation.
See OpenAI's [API Documentation](https://platform.openai.com/docs/api-reference/completions/create#completions/create-stream)

**Reply by levid0s** [reply]
I think you're right, I can see `stream` hardcoded to `True`. It would be nice to make it customizable.

https://github.com/TheR1D/shell_gpt/blob/main/sgpt/client.py#L54

**Reply by TheR1D** [reply]
If you need a quick solution: `echo $(sgpt "test")`.

**Comment by danarth** [comment]
I raised #289 to add a config option to disable streaming
